trlx/trainer/accelerate_base_trainer.py

```diff
@@ -129,6 +129,19 @@ def __init__(self, config, **kwargs):  # noqa: C901
                     "Set `tracker` to `None` to disable tracking."
                 )
 
+        self.setup_generate_sweep_kwarg()
+        self.nth_evaluation = 0
+
+    def setup_generate_sweep_kwarg(self):
```

Comment:
let's drop this function definition, since it's really single purpose and cannot be used elsewhere, and keep the code block below as is

Reply:
> let's drop this function definition, since it's really single purpose and cannot be used elsewhere, and keep the code block below as is

makes sense